docker/buildkite/Dockerfile

```diff
@@ -8,3 +8,18 @@ ADD go.mod go.sum /go/src/go.uber.org/cadence/
 # allow git-status and similar to work
 RUN git config --global --add safe.directory /go/src/go.uber.org/cadence
 RUN GO111MODULE=on go mod download
+
+# Install dependencies needed to install buildkite-agent
+RUN apt-get install -y apt-transport-https dirmngr curl
+
+# Add Buildkite's GPG key
+RUN curl -fsSL https://keys.openpgp.org/vks/v1/by-fingerprint/32A37959C2FA5C3C99EFBC32A79206696452D198 | \
```

Comment:
may I ask as to the source of this? Is his on BK's docs somewhere?

Reply:
ah, yeah, ok https://buildkite.com/docs/agent/v3/ubuntu

Reply:
Let me add that as a comment